src/main/java/org/opensearch/neuralsearch/processor/combination/GeometricMeanScoreCombinationTechnique.java

```diff
@@ -9,15 +9,18 @@
 import java.util.Set;
 
 import lombok.ToString;
+import org.opensearch.neuralsearch.processor.explain.ExplainableTechnique;
+
+import static org.opensearch.neuralsearch.processor.combination.ScoreCombinationUtil.PARAM_NAME_WEIGHTS;
+import static org.opensearch.neuralsearch.processor.explain.ExplanationUtils.describeCombinationTechnique;
 
 /**
  * Abstracts combination of scores based on geometrical mean method
  */
 @ToString(onlyExplicitlyIncluded = true)
-public class GeometricMeanScoreCombinationTechnique implements ScoreCombinationTechnique {
+public class GeometricMeanScoreCombinationTechnique implements ScoreCombinationTechnique, ExplainableTechnique {
     @ToString.Include
     public static final String TECHNIQUE_NAME = "geometric_mean";
-    public static final String PARAM_NAME_WEIGHTS = "weights";
```

Comment:
Why we removed this?

Reply:
it is a duplicate, same constant is in ScoreCombinationUtil, we reference it from all technique classes https://github.com/martin-gaievski/neural-search/blob/poc/explain_for_hybrid_v2/src/main/java/org/opensearch/neuralsearch/processor/combination/GeometricMeanScoreCombinationTechnique.java#L14